src/common/marlin_client.c

```diff
@@ -49,6 +49,7 @@ typedef struct _marlin_client_t {
     uint32_t command;            // processed command (G28,G29,M701,M702,M600)
     marlin_host_prompt_t prompt; // current host prompt structure (type and buttons)
     uint8_t reheating;           // reheating in progress
+    dialog_cb_t dialog_cb;
```

Comment:
document the meaning and usage of this var

Reply:
comment added